server/views/pages/contact-us.ejs

```diff
@@ -0,0 +1,32 @@
+<% include ../helpers/helpers %>
+
+<!DOCTYPE html>
+<html lang="en" class="govuk-template">
+  <head>
+    <% include ../partials/head %>
+  </head>
+  <body class="govuk-template__body">
+    <% include ../partials/header %>
+
+    <div class="govuk-width-container">
+      <% include ../partials/beta-banner %>
+      <a href="javascript:history.back()" class="govuk-back-link">Back</a>
```

Comment:
I've not done vanilla routing in a long time, is this normal?

Reply:
A couple of ways of doing this. I went with this because its less messy in the code and generally easiest if a little _less_ accessible. Two other possibilities. Add script blocks and `addEventListener` for `click` and `keypress` and go to the referrer url (or do a browser back like above or pass the last url about as a session var). Or we could again have script blocks and `document.write` in the _back link_ with the referrer url, which gives the best view from an accessibility perspective, but is a bit horrible from a _Javascript writing content into a static page_ perspective. 
The issue with using the `referrer` is that if you click the contact us link again at the bottom you enter into a back loop, so we'd probably have to disable the contact us link in the footer template for this page as well.
Those are your options. I might have missed one? What were you thinking?

Reply:
Can we just link this back to / and take the javascript out, that's what the links in the footer of gov.uk do

Reply:
I'm going to raise a ticket /jfdi that the banner text links to / not # too

Reply:
I'll change to `/`, it means back isn't truly a back, but it does alleviate the `history.back` which theoretically could be utilised as part of a phish.